Check if container already exits

ludojmj opened the pull request:
The container may have already been created during a previous release.
Hence the script "create-container.ps1" must first check if the container exists in the storage account to avoid throwing an exception and therefore interrupt a new release process.

ludojmj committed:
Check if container already exits

The container may have already been created during a previous release.
So the script must check the container existence first to avoid throwing an exception.

carlosjdelgado commented:
Thanks @ludojmj !

I think this should be configurable inside the task, a checkbox that defines if task will be skipped when container exists for cover all needs.

Could you implement it? It should not be very complex.

ludojmj committed:
Dev (#1)

* Checkbox to skip task when container exists

Adding a checkbox that defines if task will be skipped when container exists

* Checkbox to skip task when container exists

Adding a checkbox that defines if task will be skipped when container exists

* Typo

Adding a checkbox that defines if task will be skipped when container exists

* Typo

Adding a checkbox that defines if task will be skipped when container exists

* Split test ContinueOnExistence and TestContainer

* Adding param $ContinueOnExistence

ludojmj commented:
Hello @carlosjdelgado 
I hope this implementation is what you were waiting for.

carlosjdelgado commented on AzureStorageContainer/task.json:
```diff
@@ -57,7 +57,14 @@
+      "type": "boolean",
+      "label": "Continue on container existence",
+      "defaultValue": false,
+      "required": false,
```
Could you set this as required like in the PS script?

carlosjdelgado reviewed (commented):
It seems correct, but I need some time for test the extensión before publish It. I hope during week. 

Thanks for contributing!

ludojmj committed:
Set input "ContinueOnExistence" as required like in the PS

ludojmj commented:
Done.
